packages/runtime/src/components/builtin/Root/Root.tsx

```diff
@@ -16,7 +12,7 @@ import { useStyle } from '../../../runtimes/react/use-style'
 
 type Props = {
   children?: GridValue
-  backgrounds?: BackgroundsValue
+  backgrounds?: BackgroundsPropControllerData
```

Comment:
This should be decoupled from the prop controller data

Reply:
I discarded the changes related to the Background. Let's focus on Link for now.